Components/Badges/badges.html

```diff
@@ -1,47 +1,143 @@
-        </div>
-        <div class="badges badge-button">
-          <span>10</span>
-          <button>Button</button>
-        </div>
-        <div class="badges badge-bell">
-          <i class="far fa-bell"></i><span>10</span>
-        </div>
-        <div class="badges badge-cart">
-          <i class="fas fa-shopping-cart"></i><span>10</span>
-        </div>
-           <div class="avatar-badge avatar-active">
-             <img src="assets/avatar.jpg" alt="avatar" class="avatar-badge-img">
-             <div></div>
-           </div>
-           <div class="avatar-badge avatar-live">
-            <img src="assets/avatar.jpg" alt="avatar" class="avatar-badge-img">
-            <div></div>
+    <script
+      src="https://kit.fontawesome.com/cb9e475ff7.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <main class="documentation">
+      <header>
+        <nav class="navigation">
+          <a href="/index.html" class="logo"
+            >MindifyUI<img src="/assets/logo.svg" alt="logo" class="img-logo" />
+          </a>
+          <div class="secondary-link">
+            <a href="#" class="secondary-logo"><i class="fab fa-github"></i></a>
+            <a href="#" class="secondary-logo"><i class="fab fa-twitter"></i></a>
+            <a href="#" class="secondary-logo"><i class="fab fa-linkedin-in"></i></a>
           </div>
-          <div class="avatar-badge avatar-offline">
-            <img src="assets/avatar.jpg" alt="avatar" class="avatar-badge-img">
-            <div></div>
+        </nav>
+      </header>
+      <hr />
+      <section class="content">
+        <nav class="hamburger-menu">
+          <button aria-expanded="false" aria-controls="list">
+            <span class="open">☰</span>
+          </button>
+          <ul class="list">
+            <li><a href="#">INSTALLATION</a></li>
+            <li><a href="/Components/alert/alert.html">ALERT</a></li>
+            <li><a href="/Components/avatar/avatar.html">AVATAR</a></li>
+            <li><a href="/Components/Badges/badges.html">BADGES</a></li>
+            <li><a href="/Components/Buttons/buttons.html">BUTTONS</a></li>
+            <li><a href="/Components/card/card.html">CARD</a></li>
+            <li><a href="/Components/Images/images.html">IMAGE</a></li>
+            <li><a href="/Components/input/input.html">INPUT</a></li>
+            <li><a href="/Components/list/list.html">LIST</a></li>
+            <li><a href="/Components/modal/modal.html">MODAL</a></li>
+            <li><a href="/Components/text-utilities/text-utilities.html">TEXT-UTILITIES</a></li>            
+            <li><a href="/Components/navigation/navigation.html">NAVIGATION</a></li>
+            <li><a href="/Components/ratings/ratings.html">RATINGS</a></li>
+            <li><a href="/Components/slider/slider.html">SLIDER</a></li>              
+          </ul>
+        </nav>
+        <div class="sidebar-wrapper">

```

Comment:
You can use semantic html like aside or nav for the sidebar.

Reply:
I have use navbar many place it creates confusion for me to identify

Reply:
Cool, you can merge Archit.